src/main/webapp/app/overview/course-lectures/course-lecture-details.component.scss

```diff
@@ -0,0 +1,9 @@
+.wrapper-btn-manage-lecture {
+    display: flex;
+    align-items: center;
+}
+
+.btn-manage-lecture {
+    height: fit-content;
```

Comment:
When I remove this line of CSS in the browser DevTools, I don't see a change. Is this needed for some edge case in the layout?